### ISSUE TITLE
Import Not Working

### ISSUE THREAD

**Issue by coreystokes-8451** [databrickslabs/databricks-sync#104]
Hello,

I keep receiving the following error when trying to use import:

```
INFO] ╷
2022-02-22 11:15:05 [INFO] │ Error: Invalid index
2022-02-22 11:15:05 [INFO] │ 
2022-02-22 11:15:05 [INFO] │   on databricks_group_admins_members.tf.json line 6, in locals.databricks_group_members_databricks_group_admins_members_for_each_var.service-principal-0e409824-d286-49ff-aaf5-5eb65156a5ab:
2022-02-22 11:15:05 [INFO] │    6:                 "member_id": "${\"0e409824-d286-49ff-aaf5-5eb65156a5ab\" == var.ME_USERNAME ? \"something temp will be skipped\" : databricks_user.databricks_scim_users[\"0e409824-d286-49ff-aaf5-5eb65156a5ab\"].id}"
2022-02-22 11:15:05 [INFO] │     ├────────────────
2022-02-22 11:15:05 [INFO] │     │ databricks_user.databricks_scim_users is object with 6 attributes
2022-02-22 11:15:05 [INFO] │ 
2022-02-22 11:15:05 [INFO] │ The given key does not identify an element in this collection value.
```

Here are the error when i just try to import policies only:

```
╷
2022-02-22 11:16:51 [INFO] │ Error: Reference to undeclared resource
2022-02-22 11:16:51 [INFO] │ 
2022-02-22 11:16:51 [INFO] │   on databricks_cluster_policy_C96203490C00012A_permissions.tf.json line 8, in resource.databricks_permissions.databricks_cluster_policy_C96203490C00012A_permissions.depends_on:
2022-02-22 11:16:51 [INFO] │    8:                     "databricks_user.databricks_scim_users",
2022-02-22 11:16:51 [INFO] │ 
2022-02-22 11:16:51 [INFO] │ A managed resource "databricks_user" "databricks_scim_users" has not been
2022-02-22 11:16:51 [INFO] │ declared in the root module.
```

**Comment by stikkireddy** [comment]
@coreystokes-8451 thanks for posting this, it looks like you have some policies attached to service principals, the permissions flow thinks its a user. I will try to push a fix for this tomorrow.

**Comment by stikkireddy** [comment]
closing as issue has been resolved by #105
